src/web/event-notifications/event-notification-details/SlackNotificationDetails.jsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+import {ReadOnlyFormGroup} from 'components/common';
+import {Well} from 'components/graylog';
+import styles from './SlackNotificationSummary.css';
+
+
+const SlackNotificationDetails = ({ notification }) => (
+  <>
+    <ReadOnlyFormGroup label="Webhook URL" value={notification.config.webhook_url} />
+    <ReadOnlyFormGroup label="Channel" value={notification.config.channel} />
+    <ReadOnlyFormGroup label="Custom Message Template "
+                       value={(
+                         <Well bsSize="small" className={styles.bodyPreview} >
+                           {notification.config.custom_message || <em>Empty body</em>}
+                         </Well>
+                       )} />
+      <ReadOnlyFormGroup label="User Name" value={notification.config.username} />
+      <ReadOnlyFormGroup label="Notify Channel" value={notification.config.notify_channel} />
+      <ReadOnlyFormGroup label="Link Names" value={notification.config.link_names} />
```

Comment:
```suggestion
      <ReadOnlyFormGroup label="Link Names" value={notification.config.link_names ? `Yes` : `No`} />
```

Reply:
![notify_channel_image](https://user-images.githubusercontent.com/6947168/98038942-8b7def00-1de3-11eb-8771-f207d59179f8.png)

adding the ternary operator removes the icon, with this commit suggestion. please advice, if you would like to remove this behavior.

Reply:
Did not realize `ReadOnlyFormGroup` had that functionality built in.  Ignore my suggestion for ternary operator.